testing/python/fixtures.py

```diff
@@ -5009,3 +5009,27 @@ def test_result():
     )
     result = pytester.runpytest()
     assert result.ret == 0
+
+
+def test_parametrized_fixture_scope_allowed(pytester: Pytester) -> None:
+    """
+    Make sure scope from parametrize does not affect fixture's ability to be
+    depended upon.
+
+    Regression test for #13248
+    """
+    pytester.makepyfile(
+        """
+        import pytest
+
+        @pytest.fixture(scope="session")
+        def my_fixture(request):
+            return getattr(request, "param", None)
+
+        @pytest.mark.parametrize("my_fixture", ["a"], indirect=True, scope="function")
```

Comment:
this should always fail as the scope of the fixture is not under control of parameterize